tests/test_mock_dream.py

```diff
@@ -96,51 +95,57 @@ async def test_exit(self):
         await asyncio.sleep(0.5)
         self.assertFalse(self.srv.connected)
 
-    async def verify_command(self, command, parameters=None):
+    async def verify_command(self, action, **kwargs):
         self.assertTrue(self.srv.connected)
-        cmd_id = 1
-        time_command_sent = time.time()
-        if parameters:
-            await self.write(
-                command=command,
-                cmd_id=cmd_id,
-                time_command_sent=time_command_sent,
-                parameters=parameters,
-            )
-        else:
-            await self.write(
-                command=command,
-                cmd_id=cmd_id,
-                time_command_sent=time_command_sent,
-            )
+        request_id = 1
+        await self.write(
+            action=action,
+            request_id=request_id,
+            **kwargs,
+        )
         # Give time to the socket server to process the command.
         await asyncio.sleep(0.5)
         data = await self.read()
-        self.assertEqual(data["cmd_id"], cmd_id)
-        self.assertGreater(data["time_command_received"], time_command_sent)
-        self.assertGreater(data["time_ack_sent"], data["time_command_received"])
-        self.assertEqual(data["response"], "OK")
+        self.assertEqual(data["request_id"], request_id)
+        self.assertEqual(data["result"], "ok")
+
+    async def verify_error(self, action, **kwargs):
+        self.assertTrue(self.srv.connected)
+        request_id = 1
```

Comment:
I would prefer to use some other numbers than pure 1. One of the solution I found:

```
import random
....
random.seed(42) # or any preferred number
```

and then:

```
       request_id = random.randint(2, 5000)
 ```